server/src/api/merchants.ts

```diff
@@ -18,10 +18,24 @@
  * @fileoverview Handles routing of /merchants endpoints.
  */
 
-import express from 'express';
+import {Router, Request, Response, NextFunction} from 'express';
 
+import {MerchantPayload} from '../interfaces';
 import {merchantService} from '../services';
 
-const router: express.Router = express.Router();
+const merchantRouter: Router = Router();
 
-export const merchantRouter: express.Router = router;
+merchantRouter.post(
+  '/',
+  async (req: Request, res: Response, next: NextFunction) => {
+    try {
+      const merchant: MerchantPayload = req.body;
+      await merchantService.addMerchant(merchant);
+      res.sendStatus(200);
```

Comment:
No need to send back merchant id?

Reply:
No need. The frontend has `idToken` from Firebase Authentication. They should attach it as Authorization header to the backend. This will the one used to identify the merchant.

Reply:
Okay, based on discussion on Chat, will send id back to merchant.